api/v1beta1/access_control_validate.go

```diff
@@ -68,6 +75,12 @@ var Privileges = map[string][]PrivilegeScope{
 	"user-admin":     {Global},
 }
 
+var Post6Privileges = map[string][]PrivilegeScope{
```

Comment:
I think Privileges should be all privileges including 6.0 privileges. Validation should fail if version < 6.0 and post6.0 privilege is configured. Simpler is code to use Privileges as being the complete set.